src/v2/controller/auth.controller.ts

```diff
@@ -0,0 +1,91 @@
+import { Request as ExRequest, Response as ExResponse } from "express";
+import { Body, Controller, Get, Path, Post, Query, Request, Route, Tags } from "tsoa";
+import AuthService from "../service/auth.service";
+
+@Route("auth")
+@Tags("Auth")
+export class AuthController extends Controller {
+	private readonly service = new AuthService();
+
+	/**
+	 * Handles Discord authorization page redirect
+	 */
+	@Get("discord/callback")
```

Comment:
shouldn't this be authorized? I feel like having the tokens already present here could be prone to abuse